app/models/records.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+class DidLogRecord(BaseModel):
+    scid: str = Field()
+    logHistory: str = Field()
```

Comment:
Is this camel case intended for the response values or something? pythonic convention is snake case.